pkg/placement/membership.go

```diff
@@ -335,21 +338,40 @@ func (p *Service) performTableDissemination(ctx context.Context) error {
 	p.disseminateLock.Lock()
 	defer p.disseminateLock.Unlock()
 
-	state := p.raftNode.FSM().PlacementState()
-
-	log.Infof(
-		"Start disseminating tables. memberUpdateCount: %d, streams: %d, targets: %d, table generation: %s",
-		cnt, nStreamConnPool, nTargetConns, state.GetVersion())
 	p.streamConnPoolLock.RLock()
 	streamConnPool := make([]placementGRPCStream, len(p.streamConnPool))
 	copy(streamConnPool, p.streamConnPool)
+
```

Comment:
This is more what I meant.

```suggestion
	req := &placementTablesUpdate{
		hosts: streamConnPool,
	}

	for _, stream := range streamConnPool {
		if hostAcceptsVNodes(stream) {
			if req.wtihVNodes == nil {
				req.withVNodes = fsm.PlacementState(false)
			}
		} else {
			if req.noVNodes == nil {
				req.noVNodes = fsm.PlacementState(false)
			}
		}

		if req.withVNodes != nil && req.noVNodes != nil {
			break
		}
	}
 
```